metadata.json

```diff
@@ -15,6 +15,10 @@
     "alias"
   ],
   "dependencies": [
+     {
```

Comment:
Yes, it has to be for the functions to be loaded; dependency in metadata json is used by DependencyLoader when loader.load(:function, "libkv::something") is called by call_function